.github/workflows/publish.yml

```
@@ -0,0 +1,31 @@
+name: publish
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - v*
```

I think this needs to be quoted? 
```suggestion
  push:
    tags:
      - 'v*'
```

---

hm we use a similar publish.yml here that seems to work - https://github.com/maplibre/maplibre-gl-geocoder/blob/main/.github/workflows/publish.yml